server/routes/index.ts

```diff
@@ -0,0 +1,9 @@
+import { Router as createRouter } from "express";
+import type { Router } from "express";
+import { forumRouter } from "./forum";
+import { settingsRouter } from "./settings";
+
+export const apiRouter: Router = createRouter();
+
+forumRouter(apiRouter);
```

Comment:
Я бы лучше возвращал из файлов роутеры и прицеплял их к `apiRouter` прямо тут

```suggestion
 apiRouter
  .use("/settings", settingsRouter)
  .use("/forum", forumRouter);
```

Такое поведение более явное, чем передача роутера параметром и его изменение внутри функции